catalog/plugin/catalog-plugin-query-monitor-commands/src/main/java/ddf.catalog.querymonitorcommands/QueryMonitorCommandPrintTable.java

```diff
@@ -0,0 +1,187 @@
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.querymonitorcommands;
+
+import java.io.PrintStream;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.UUID;
+
+import org.apache.karaf.shell.api.action.Action;
+import org.apache.karaf.shell.api.action.Command;
+import org.fusesource.jansi.Ansi;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.querymonitor.ActiveSearch;
+import ddf.catalog.querymonitor.QueryMonitorPlugin;
+
+@Command(scope = "catalog", name = "printtable", description = "Sets flag to determine whether to delete active searches from table once they complete")
+public class QueryMonitorCommandPrintTable implements Action {
+
+    private static final Logger LOGGER =
+            LoggerFactory.getLogger(QueryMonitorCommandPrintTable.class);
+
+    QueryMonitorPlugin queryMonitor1 = null;
+
+    private static final int USER_MAX_LENGTH = 20;
+
+    private static final int SOURCE_MAX_LENGTH = 20;
+
+    private static final int SEARCH_MAX_LENGTH = 33;
+
+    private static final int TIME_MAX_LENGTH = 30;
+
+    private static final int UUID_MAX_LENGTH = 37;
+
+    private String formatString =
+            "%1$-" + USER_MAX_LENGTH + "s %2$-" + SOURCE_MAX_LENGTH + "s %3$-" + SEARCH_MAX_LENGTH
+                    + "s %4$-" + TIME_MAX_LENGTH + "s %5$-" + UUID_MAX_LENGTH + "s %n";
+
+    PrintStream console = null;
+
+    PrintStream getPrintStream() {
+        return System.out;
+    }
+
+    public QueryMonitorCommandPrintTable(QueryMonitorPlugin q) {
+        queryMonitor1 = q;
+        console = getPrintStream();
+    }
+
+    public QueryMonitorCommandPrintTable() {
+        console = getPrintStream();
+    }
+
+    /**
+     * Method to implement {@link Action}. Called when the catalog:printtable command is called by the shell.
+     * Prints out a table representing a {@link QueryMonitorPlugin}'s ActiveSearchTable including information
+     * on all {@link ActiveSearch}'s currently populating the table
+     *
+     * @return always returns {@code null}
+     */
+
+    @Override
+    public Object execute() throws Exception {
+        try {
+            printSearchTableToConsole();
+        } catch (Exception e) {
+            LOGGER.debug(
+                    "Exception encountered in doExecute of QueryMonitorCommandPrintActiveSearcHTable.java!"
+                            + e.getMessage());
```

Comment:
Instead of `+ e.getMessage()` just do `, e`. Unless we're intentionally hiding stack trace